Static Framework Sample/README.md

```diff
@@ -0,0 +1,32 @@
+# 🎉 Static Framework iOS sample 
+
+### This iOS application will show you how to use Contentsquare SDK with Static framework dependency. <br />
+
+## Prerequisites
+
+Go to [Contentsquare github](https://github.com/ContentSquare/CS_iOS_SDK/releases), under Assets you should be able to find `ContentsquareModuleDynamicManually.xcframework.zip` file, please download it.
+
+Once you open the zip file on your computer, you should have two directories named `ContentsquareModule.xcframework` and `SwiftProtobuf.xcframework`.
+
+Copy both of them and paste it inside `ContentsquareModule` folder of the [current project](https://github.com/ContentSquare/iOS-sample-app/tree/mobile-4115/Improve-sample-apps/Static%20Framework%20Sample/ContentsquareModule).
```

Comment:
I think git doesn't keep track of folders but only files.
So if someone does a first checkout of this repo, he won't have a `ContentsquareModule` folder so the instructions will quite misleading.
Maybe change the framework location at the root level ?

Reply:
Indeed Git doesn't keep track of empty folders, but I added a .gitkeep inside the folder so when you clone it you still have it. If you think it's not a good idea and its better to put it at the root level, I will change it? :)

Reply:
Well done, I didn't see it, good for me.